templates/package.hamlet

```diff
@@ -17,7 +17,8 @@ $newline never
             <h1>
                #{pn} #
                <span .latest-version>
-                 #{displayedVersion} #
+                 <a href="https://hackage.haskell.org/package/#{pn}">
```

Comment:
@snoyberg okay I moved the latest Hackage version to below the Stackage version lines.

@DanBurton I am thinking to add a separate package history page later when I find the time.